assets/js/blocks/cart-checkout/cart/inner-blocks/filled-cart-block/edit.tsx

```diff
@@ -35,19 +35,17 @@ export const Edit = ( { clientId }: { clientId: string } ): JSX.Element => {
 			{ ...blockProps }
 			hidden={ currentView !== 'woocommerce/filled-cart-block' }
 		>
-			<Columns>
```

Comment:
@senadir these were causing double block wrappers. They don't seem to do anything.